guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
```

Comment:
This is not the only way to end up with a multi-environment host. You can also (re)assign a host to multiple content view environments after registration, as described elsewhere in this PR.

```suggestion
Have at least one multi-environment host. 
```